### ISSUE TITLE
Vocab.SRC file not found

### ISSUE THREAD

**Issue by shantipriyap** [shantipriyap/OdiaGenAI#13]
![BCAAD59D-3394-4FCA-8BA3-A97661065C83_4_5005_c](https://user-images.githubusercontent.com/33823853/233352546-ec7a6b09-1e84-43fb-b9c4-c7478fee5962.jpeg)

Vocab-SRC file not found error. Please check the attached snapshot.

**Comment by swateek** [comment]
For a workaround, please use the below commands, in the PROJECT home directory.

```bash
curl https://ai4b-public-nlu-nlg.objectstore.e2enetworks.net/en2indic.zip  en2indic.zip
unzip en2indic.zip
```

This just needs to be run once.

**Comment by swateek** [comment]
This has been addressed via pull request #14 created for issue #12
